tests/single-node-sample-apps/test-rdma.sh

```diff
@@ -136,14 +144,14 @@ function run_test_rdma() {
     info "Connection type: rdma"
 
     info "Starting Tx side media_proxy"
-    local tx_media_proxy_cmd="media_proxy -i 192.168.96.1 -t 8002"
+    local tx_media_proxy_cmd="media_proxy -t 8002"
     run_in_background "$bin_dir/$tx_media_proxy_cmd" "$tx_media_proxy_out"
     tx_media_proxy_pid=$!
 
     sleep 1
 
     info "Starting Rx side media_proxy"
-    local rx_media_proxy_cmd="media_proxy -i 192.168.96.2 -t 8003"
+    local rx_media_proxy_cmd="media_proxy -t 8003"
     run_in_background "$bin_dir/$rx_media_proxy_cmd" "$rx_media_proxy_out"
     rx_media_proxy_pid=$!
```

Comment:
Hey I am not sure what you wanted to achieve but this is totally wrong. This way media proxy will default to predefined parameters and the MTL will be initialized for them as well. Currently the parameters that are mandatory are `-d` and `-i` and probably this should be reflected in not using defaults but exiting with error:

![image](https://github.com/user-attachments/assets/9b52e0b1-87df-4903-b5ee-2bbd029cd9ef)

Reply:
this is only for RDMA. This test don't create any MTL session. The parameters are not used, so I did not pass any